idaes/core/scaling/custom_scaler_base.py

```diff
@@ -461,6 +461,69 @@ def scale_variable_by_units(self, variable, overwrite: bool = False):
                 "in self.unit_scaling_factors"
             )
 
+    def _build_replacement_map(self, variable, constraint, safety_mode: bool):
+        """
+        Helper function for scale_variable_by_definition_constraint.
+
+        Walks the constraint to build a map between Variables or Expressions
```

Comment:
This doesn't _sound_ right to me. What is meant by "walks"? A synonym for "helps"?